### PR TITLE
[Mobile app] - hotfix search feature

### PR DESCRIPTION
**Pull request by NoahNsimbe**
#### Summary of Changes (What does this PR do?)

- Fixes failure of the search feature.

#### Status of maturity (all need to be checked before merging):

- [ ] I've tested this locally
- [ ] I consider this code done
- [ ] This change ready to hit production in its current state
- [ ] The title of the PR states what changed and the related issues number (used for the release note).
- [ ] I've included issue number in the "Closes #ISSUE-NUMBER" part of the "[What are the relevant tickets?](#what-are-the-relevant-tickets)" section to [link the issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).
- [ ] I've updated corresponding documentation for the changes in this PR.
- [ ] I have written unit and/or e2e tests for my change(s).

#### How should this be manually tested?

- Please include the steps to be done inorder to setup and test this PR.

#### What are the relevant tickets?

- Closes #<enter issue number here>
- [Jira_card_number]() (If exists)

### CODE REVIEW

File: mobile/packages/airqo_api/lib/src/models/prediction.dart [hunk 1]
```diff
@@ -35,8 +35,7 @@ class Prediction {
   static List<Prediction> parsePredictions(Map<String, dynamic> jsonBody) {
     final suggestions = <Prediction>[];
 
-    final jsonArray = jsonBody['predictions'];
-    for (final jsonElement in jsonArray as List<Map<String, dynamic>>) {
```
Comment (NoahNsimbe) [inline mobile/packages/airqo_api/lib/src/models/prediction.dart:39]: Removed casting which is causing failure of search feature

File: mobile/lib/models/models.dart [hunk 2]
```diff
@@ -10,4 +10,3 @@ export 'kya.dart';
 export 'notification.dart';
 export 'place_details.dart';
 export 'profile.dart';
-export 'suggestion.dart';
```
Comment (NoahNsimbe) [inline mobile/lib/models/models.dart:13]: Decluttering the codebase. Removing unused files and classes.

File: mobile/packages/airqo_api/lib/src/models/prediction.dart (hunk 1 above)
Comment (Baalmart) [reply mobile/packages/airqo_api/lib/src/models/prediction.dart:39]: Thanks @NoahNsimbe

**Review by Baalmart — APPROVED** [review]
Thanks @NoahNsimbe , LGTM

**Pull request merged**